components/SingleReport.vue

```diff
@@ -36,22 +34,22 @@ const createLabelsFromCriteria = (currentCriterion) => {
     <div class="py-6 px-4 sm:px-6 lg:px-8">
       <div>
         <div class="flex justify-between items-center gap-6">
-          <h1 class="mb-0 break-all">{{ props.report.name }}</h1>
-          <AxoLink target="_blank" :href="sourcegraphUrl"
+          <h1 class="mb-0 break-all">{{ report?.name }}</h1>
```

Comment:
Right, because in the production code, it's a required prop. That might still end up being corrupted and printing undefined here, but that's a third-level bug so I think we can leave it for not. 😅